bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractUserTaskBuilder.java

```diff
@@ -156,16 +156,20 @@ public B zeebeFormId(final String formId) {
     return myself;
   }
 
-  /**
-   * Marks the user task as native Zeebe user task.
-   *
-   * @return the builder object
-   */
+  @Override
   public B zeebeUserTask() {
     getCreateSingleExtensionElement(ZeebeUserTask.class);
     return myself;
   }
 
+  @Override
+  public B zeebeExternalReference(final String externalReference) {
```

Comment:
:upside_down_face: ~Same as in the `UserTaskRecord`, would this method be more readable if it was named `zeebeExternalFormReference`?~

Nvm, I just saw that it's an attribute of the `formDefinition` element, so it provides context and the above isn't necessary.

Reply:
Yeah, but the `formId` and `formKey` are also not named `id` or `key`, so 🤷

Reply:
Renamed it in the builder at least (not in the model itself).